.travis.yml

```
@@ -6,8 +6,11 @@ cache:
   - node_modules
 install:
 - npm install
+- npm install -g codecov
 script:
 - npm run build
+- jest --coverage
+- codecov
```

```suggestion
- npx codecov
```

建议使用 `npx` ，并移除 全局安装 (`npm install -g codecov`) 这一步骤，以减少上下文之间的关联